src/main/java/com/cloudbees/jenkins/support/SupportPlugin.java

```diff
@@ -563,6 +561,46 @@ public void run() {
         return Collections.emptyList();
     }
 
+    /**
+     * Returns the full bundle name.
+     *
+     * @return the full bundle name.
+     */
+    @NonNull
+    public static String getBundleFileName() {
+        StringBuilder filename = new StringBuilder();
+        filename.append(getBundlePrefix());
+
+        SimpleDateFormat dateFormat = new SimpleDateFormat("yyyy-MM-dd_HH.mm.ss");
+        dateFormat.setTimeZone(TimeZone.getTimeZone("GMT"));
+        filename.append("_").append(dateFormat.format(new Date()));
+
+        filename.append(".zip");
```

Comment:
so, finally used `filename` directly instead of a temp file created by the dedicated API.